src/assets/scripts/client/game/GameOptions.js

```diff
@@ -34,22 +58,25 @@ export default class GameOptions {
     /**
      * @for GameOptions
      * @method addOption
+     * @param optionProps {object}
      */
-    addOption(data) {
-        const optionStorageName = `zlsa.atc.option.${data.name}`;
-        this._options[data.name] = data;
-        let dataName = data.defaultValue;
+    addOption(optionProps) {
+        const optionStorageKey = this.buildStorageName(optionProps.name);
+        const storedOptionValue = global.localStorage.getItem(optionStorageKey);
+        this._options[optionProps.name] = optionProps;
+        let optionValue = optionProps.defaultValue;
 
-        if (_has(localStorage, optionStorageName)) {
-            dataName = localStorage[optionStorageName];
+        if (typeof storedOptionValue !== 'undefined') {
```

Comment:
just curious, why aren't we using `!_isNil()` here?

Reply:
It's more explicit. LocalStorage will return `undefined` when a key does not exist.   I suppose I could switch it, probably should to stay consistent.